scripts/generate-pr-changelogs.sh

```diff
@@ -72,7 +73,7 @@ select_notable() {
 temp_changelog_yaml="$(mktemp)"
 
 for pr_number in $(
-      git log --merges --oneline --ancestry-path "$commit_range" \
+    git log --merges --oneline --ancestry-path "$(git merge-base --fork-point origin/HEAD "$commit_since")".."$commit_to" \
```

Comment:
I wonder whether it would make more sense to use `$commit_to` instead of `origin/HEAD`

Reply:
Good idea. I'll try it. But it seems my current solution does not work so I'll have to change it.